assets/tests/QuestionSchemaValidationService.js

```diff
@@ -0,0 +1,25 @@
+// Copyright 2017 The TIE Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Service for validating data put into a question,
+ * before that schema is committed.
+ */
+
+// TODO(eyurko): Add functions to validate question schema.
+tie.factory('QuestionSchemaValidationService', [
```

Comment:
Suggest just dropping this whole file until the next PR; let's try to keep the codebase free of dead code (even for a short period of time).